src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx

```diff
@@ -46,13 +46,9 @@ Keep reading for details about New Relic One pricing and billing.
 
 ## Billing and usage in the UI [#billing-usage-ui]
 
-The ability to edit billing-related settings depends on user permissions. For users on the New Relic One user model, this requires the [**Billing user**](https://docs.newrelic.com/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure#roles) role. For users on the original user model, this requires an [Owner or Admin role](https://docs.newrelic.com/docs/accounts/original-accounts-billing/original-users-roles/users-roles-original-user-model#roles). 
+The ability to edit billing-related settings depends on user permissions. For users on the New Relic One user model, this requires the [**Billing user**](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure#roles) role. For users on the original user model, this requires an [Owner or Admin role](/docs/accounts/original-accounts-billing/original-users-roles/users-roles-original-user-model#roles). 
 
-To view and manage billing and usage in the UI:
-
-* For billing details, including credit card information: from the [account dropdown](/docs/using-new-relic/welcome-new-relic/get-started/glossary#account-dropdown), click **Manage your plan** and then **Billing and invoices**.
-* For usage information: from the [account dropdown](/docs/using-new-relic/welcome-new-relic/get-started/glossary#account-dropdown), click **View your usage**. 
-* For data ingest and retention details: from the [account dropdown](/docs/using-new-relic/welcome-new-relic/get-started/glossary#account-dropdown), click **Manage your data**.
+For how to view and manage billing and usage in the UI, see [Pricing and billing UI](/docs/accounts/accounts-billing/general-account-settings/introduction-account-settings/#pricing).
```

Comment:
What if we front-loaded that link: "See Pricing and billing UI for how to view and manage your billing and usage."?

Reply:
i tend to like to put the goal/desire first cause it makes scanning starts of sentence easier. not that it matters much either way dont' think